README.md

```diff
@@ -542,6 +542,12 @@ Folder in which Python test files are located - should align with Poetry's proje
 
 Default: `${project.basedir}/tests`
 
+#### cacheBuildWheelFiles ####
+
+Enables or Disables the copying of poetry build wheel files into Poetry cache.
```

Comment:
A: I'd just say "wheels" instead of "poetry build wheel files" (same for the config name).  A bit more readable and definitely something you'd expect someone using python to understand easily.

Reply:
@ewilkins-csi this is a step in the direction to make wheels available without directory calculations. It doesn't actually exercise the config in modules that need the wheel. That's next. 

Should I express that future intend at this point?

Reply:
Ah yeah it's fine to wait to add that detail until we actually implement it.